plugins/woocommerce-admin/client/homescreen/activity-panel/orders/test/index.js

```diff
@@ -32,4 +45,25 @@ describe( 'OrdersPanel', () => {
 			screen.queryByText( 'You’ve fulfilled all your orders' )
 		).toBeInTheDocument();
 	} );
+	it( 'should call activity_panel_orders_orders_begin_fulfillment when order is clicked', () => {
+		useSelect.mockReturnValue( {
+			orders: [
+				{
+					total: 123,
+					id: 1,
+					number: 1,
+				},
+			],
+			isError: false,
+			isRequesting: false,
+		} );
+		const { getByText } = render(
+			<OrdersPanel orderStatuses={ [] } unreadOrdersCount={ 1 } />
+		);
+		userEvent.click( getByText( '0 products' ) );
+		expect( recordEvent ).toHaveBeenCalledWith(
+			'activity_panel_orders_orders_begin_fulfillment',
+			{}
+		);
```

Comment:
Side note... this test shows that that it seems odd you can fulfill an order with zero products?!